src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -62,6 +62,16 @@ private protected override Task BaseVerifyWorkerAsync(
             displayTextPrefix, inlineDescription, isComplexTextEdit, matchingFilters, flags, options, skipSpeculation: skipSpeculation);
     }
 
+    private protected override Task BaseVerifyWorkerAsync(
+        string code, int position, bool usePreviousCharAsTrigger, char? deletedCharTrigger, bool? hasSuggestionItem,
+            SourceCodeKind sourceCodeKind, ItemExpectation[] expectedResults,
+        List<CompletionFilter> matchingFilters, CompletionItemFlags? flags, CompletionOptions options, bool skipSpeculation = false)
+    {
+        return base.VerifyWorkerAsync(
+            code, position, usePreviousCharAsTrigger, deletedCharTrigger, hasSuggestionItem, sourceCodeKind,
+            expectedResults, matchingFilters, flags, options, skipSpeculation);
+    }
```

Comment:
i don't understand this.  it seems to be an overrride, which just calls the base, but passes all the values along.  why have this?

Reply:
This is the implementation of `BaseVerifyWorkerAsync` calling the base `VerifyWorkerAsync`, not the same method. Since we do not override `VerifyWorkerAsync` here, we just call its base.

Reply:
@CyrusNajmabadi if this is okay can we move the PR forward?